boards/shields/mikroe_twowire_eth_click/mikroe_twowire_eth_click_arduino.overlay

```diff
@@ -0,0 +1,23 @@
+&arduino_spi {
```

Comment:
Nack on this. If there's no arduino connector, don't make up one

Reply:
@erwango - I've added the MicroE's shield in the newest version of this patch set.
I've also followed the approach from `zephyr/boards/shields/mikroe_wifi_bt_click` shield, which allows Arduino operation as well.

My goal is to provide and upstream the setup, so either me or other peers can easily test the T1S solution.

Reply:
> shield, which allows Arduino operation as well.

Ok, then we should review this as well. It won't be possible to maintain and provide support on each and every fantasy connection that one can imagine on his/her desk.

Reply:
@erwango - I would like to also bring up the issue: the need to document the way one would be able to reproduce the setup. Of course - as I did with this PR - I can add detailed description here, but after this code is pulled - the info is gone.

I would prefer to keep this information in some form. Do you have any idea of how this can be done?

Reply:
W/o better option, I'd propose 2 options:
- Move this configuration in a board.overlay file in the sample
- Document set up used for test in commit message

Reply:
I think that, I could also provide link to my github account with those configuration files.

Adding `board.overlay` to `echo_server` sample is tricky, as it will not work without the "shield" and can cause regression on the board itself (this board would not be used in this sample as it doesn't have ETH on it).

Reply:
@erwango @jukkar - then I will just add the OA_TC6 code and eth_lan865x.

For extra *.overlay and *.conf setup I will add link to my github account and provide detailed description in this PR.

Reply:
@erwango - I've removed the shield code. Could you re-review the code (as I don't have the proper re-review mark next to yours name as I had for @jukkar)?